docs/GraFx-Developers/grafx-studio/integration-overview/05-versioning/index.md

```
@@ -110,6 +110,12 @@ If you use [Studio UI](https://github.com/chili-publish/studio-ui), the pre-buil
 
 **CDN URL Structure**
 
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)